generators/app/templates/src/app/screens/InitialLoading/index.ejs

```diff
@@ -0,0 +1,57 @@
+<%_ if (features.onboarding && !features.login) { _%>import { useEffect } from 'react';<%_ } _%>
+import PropTypes from 'prop-types';
+<%_ if (features.login) { _%>
+import { useSelector } from 'react-redux';
+import withLoadable from '@components/Loadable';
+<%_ } _%>
+import Routes from '@constants/routes';
+<%_ if (features.onboarding && !features.login) { _%>
+import { setOnBoardingAccess, getOnBoardingAccess } from '@services/OnBoardingService';
+<%_ } _%>
+
+<%_ if (features.login) { _%>
+const initialLoadingSelector = state => state.auth.initialLoading;
+<%_ } _%>
+
+const InitialLoading = ({ navigation }) => {
+  <%_ if (features.login) { _%>
+  const initialLoading = useSelector(initialLoadingSelector);
+  const currentUser = useSelector(state => state.auth.currentUser);
+  if (!initialLoading) {
+    <%_ } _%>
+    <%_ if (features.onboarding && !features.login) { _%>
+  useEffect(() => {
+    const getAccess = () => getOnBoardingAccess();
+    (async () => {
+      const access = await getAccess();
+      if (!access) setOnBoardingAccess();
+      navigation.navigate(access ? Routes.App : Routes.OnBoarding);
+    })();
+  }, [navigation]);
```

Comment:
Why this `useEffect` is inside the `features.login` declaration? Can't you sort them?

Reply:
it's not, this is the condition: `if (features.onboarding && !features.login)`